docs/services/eventhub/eventhub.md

```diff
@@ -31,6 +31,228 @@ As an output of deployment, the operator stores a JSON formatted secret with the
+  template:
+    metadata:
+      labels:
+        app: eh-consumer
+    spec:
+      nodeSelector:
+        "beta.kubernetes.io/os": linux
+      containers:
+        - name: eh-consumer
+          image: abhirockzz/eh-kafka-consumer
+          env:
+            - name: EVENTHUBS_CONNECTION_STRING
+              valueFrom:
+                secretKeyRef:
+                  name: eh-secret
+                  key: primaryConnectionString
+            - name: EVENTHUBS_NAMESPACE
+              valueFrom:
+                secretKeyRef:
+                  name: eh-secret
+                  key: eventhubNamespace
+            - name: EVENTHUBS_BROKER
+              value: \$(EVENTHUBS_NAMESPACE).servicebus.windows.net:9093
+            - name: EVENTHUBS_TOPIC
+              valueFrom:
+                secretKeyRef:
+                  name: eh-secret
+                  key: eventhubName
+            - name: EVENTHUBS_CONSUMER_GROUPID
+              value: eh-aso-cg
+EOF
+```
+
+Wait for the consumer application to start
+
+```shell
+kubectl get pods -l=app=eh-consumer -w
+
+//Keep a track of the logs for the consumer app
+kubectl logs -f $(kubectl get pods -l=app=eh-consumer --output=jsonpath={.items..metadata.name})
+```
+
+You should see something similar to:
+
+```bash
+Event Hubs broker [eh-aso-ns.servicebus.windows.net:9093]
+Sarama client consumer group ID eh-aso-cg
+new consumer group created
+Event Hubs topic eh-aso-hub
+Waiting for program to exit
+Partition allocation - map[eh-aso-hub:[0 1 2]]
+```
+
+Deploy the Producer application
+
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: eh-producer
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: eh-producer
+  template:
+    metadata:
+      labels:
+        app: eh-producer
+    spec:
+      nodeSelector:
+        "beta.kubernetes.io/os": linux
+      containers:
+        - name: eh-producer
+          image: abhirockzz/eh-kafka-producer
+          env:
+            - name: EVENTHUBS_CONNECTION_STRING
+              valueFrom:
+                secretKeyRef:
+                  name: eh-secret
+                  key: primaryConnectionString
+            - name: EVENTHUBS_NAMESPACE
+              valueFrom:
+                secretKeyRef:
+                  name: eh-secret
+                  key: eventhubNamespace
+            - name: EVENTHUBS_BROKER
+              value: \$(EVENTHUBS_NAMESPACE).servicebus.windows.net:9093
+            - name: EVENTHUBS_TOPIC
+              valueFrom:
+                secretKeyRef:
+                  name: eh-secret
+                  key: eventhubName
+EOF
+```
+
+Once the producer application is up and running, the consumer should kick in, start receiving the messages and print them to the console (check the consumer app terminal)
+
+> In case you want to check producer logs: `kubectl logs -f $(kubectl get pods -l=app=eh-producer --output=jsonpath={.items..metadata.name})`
+
+### Clean-up
```

Comment:
```suggestion
### Clean-up (Optional)
```

Might be useful to suggest further things that users might want to try out with the EventHub now they have things running.